### ISSUE TITLE
Order of __sklib unique name uses documented params order not declared params order

### ISSUE THREAD

**Issue by alexcu** [splashkit/splashkit-translator#32]
```c
int foo(int a, char b)
```

Generates: `__sklib_int_char`

```c
int foo(char b, int a)
```

also generates: `__sklib_int_char` when it should generate the other way around

**Comment by macite** [comment]
To clarify...

```c
/**
 * @param a ...
 * @param b ...
 */
int foo(int a, char b)
```
Generates: __sklib_int_char

```c
/**
 * @param a ...
 * @param b ...
 */
int foo(char b, int a)
```
also generates: __sklib_int_char when it should generate the other way around

**Comment by alexcu** [comment]
Resolved with PR #33
